README.md

```diff
@@ -51,7 +46,7 @@ More information on setting up API access using tokens can be found [in the docu
 DynatraceMetricExporter exporter =
     DynatraceMetricExporter.builder()
         .setUrl("https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest")
-        .setApiToken({YOUR_API_TOKEN}) // read from environment or config
```

Comment:
I'm wondering if the curly braces could make it clearer that the API token is missing. :thinking: